reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -55,9 +57,86 @@ export default function Quota() {
     );
   }
   return (
-    <Grid columns={2}>
-      {renderPieChart("CPU", quota.cpu)}
-      {renderPieChart("HDD", quota.disk)}
-    </Grid>
+    <>
+      <Grid columns={2}>
+        {renderPieChart("CPU", quota.cpu)}
+        {renderPieChart("Disk", quota.disk)}
+      </Grid>
+      <div className={styles["quota-info"]}>
+        {getQuotaNotifications(quota, config.adminEmail).map(
+          (notification, index) => {
+            const message = (
+              <span>
+                {notification.body} Please see:{" "}
+                <a
+                  href={REANA_QUOTAS_DOCS_URL}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {REANA_QUOTAS_DOCS_URL}
+                </a>
+              </span>
+            );
+            return (
+              <div key={index}>
+                <Notification
+                  icon={notification.icon}
+                  header={notification.header}
+                  message={message}
+                  error={notification.type === "critical"}
+                  warning={notification.type === "warning"}
+                  closable={false}
+                />
+              </div>
+            );
+          }
+        )}
+      </div>
+    </>
   );
 }
+
+function getQuotaNotificationInfo(quotaType, exceededLimit, adminEmail) {
+  if (quotaType === "disk" && exceededLimit)
+    return "Please delete unnecessary workflows or files.";
+  if (quotaType === "disk" && !exceededLimit)
+    return "Please consider liberating disk space by deleting some workflows or files.";
+  if (quotaType === "cpu" && exceededLimit)
+    return (
+      <span>
+        Please contact{" "}
+        {adminEmail ? (
+          <a href={`mailto:${adminEmail}`}>REANA administrators</a>
+        ) : (
+          "REANA administrators"
+        )}{" "}
+        for quota limit increase.
+      </span>
+    );
+}
+
+export function getQuotaNotifications(quota, adminEmail) {
+  let notifications = [];
+  for (const [key, value] of Object.entries(quota)) {
+    const { usage, limit, health } = value;
+    if (health !== "critical") continue;
```

Comment:
Should we show a notification even when `health === "warning"`?

Reply:
Yes, it might be too early to generate notifications at 60%. Maybe we can continue the discussion in the [other thread](https://github.com/reanahub/reana-ui/pull/275#discussion_r892446483), given that this is quite related to that?